docs/ops/movement/ScatterElementsUpdate_12.md

```diff
@@ -0,0 +1,209 @@
+
+**Category**: *Data movement*
+
+**Short description**: Creates a copy of the first input tensor with elements updated according to the logic specified by *reduction* attribute and the following inputs: ``indices``, ``updates``, ``axis``.
+
+**Detailed description**: For each element of ``updates`` input, the value of ``indices`` at the same index points the element to update on the ``axis`` of ``data``. The index for dimension pointed by ``axis`` is provided by values of ``indices`` input, otherwise the index is the same as the index of the entry itself.
+
+The dimensions of ``update`` tensor are allowed to be less or equal to the corresponding dimensions of ``data`` tensor, but the dimension pointed by ``axis`` can be also greater (especially if the ``indices`` input contains duplicated values).
+
+The operation to perform between the corresponding elements is specified by *reduction* attribute,
+by default the elements of ``data`` tensor are simply overwritten by the values from ``updates`` input.
+
+Additionally *use_init_val* attribute can be used to control whether the elements from the ``data`` input tensor are used as initial value (enabled by default).
+
+For instance, in a 3D tensor case, the update of the element corresponding to the ``[i][j][k]`` is performed as below:
+
+- reduction == "copy"
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] = updates[i][j][k] if axis = 0,
+    output[i][indices[i][j][k]][k] = updates[i][j][k] if axis = 1,
+    output[i][j][indices[i][j][k]] = updates[i][j][k] if axis = 2
+
+
+- reduction == "sum"
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] += updates[i][j][k] if axis = 0,
+    output[i][indices[i][j][k]][k] += updates[i][j][k] if axis = 1,
+    output[i][j][indices[i][j][k]] += updates[i][j][k] if axis = 2
+
+
+- reduction == "prod"
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] *= updates[i][j][k] if axis = 0,
+    output[i][indices[i][j][k]][k] *= updates[i][j][k] if axis = 1,
+    output[i][j][indices[i][j][k]] *= updates[i][j][k] if axis = 2
+
+
+- reduction == "min"
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] = min(updates[i][j][k], output[indices[i][j][k]][j][k]) if axis = 0,
+    output[i][indices[i][j][k]][k] = min(updates[i][j][k], output[i][indices[i][j][k]][k]) if axis = 1,
+    output[i][j][indices[i][j][k]] = min(updates[i][j][k], output[i][j][indices[i][j][k]]) if axis = 2
+
+
+- reduction == "max"
+
+.. code-block:: cpp
+
+    output[indices[i][j][k]][j][k] = max(updates[i][j][k], output[indices[i][j][k]][j][k]) if axis = 0,
+    output[i][indices[i][j][k]][k] = max(updates[i][j][k], output[i][indices[i][j][k]][k]) if axis = 1,
+    output[i][j][indices[i][j][k]] = max(updates[i][j][k], output[i][j][indices[i][j][k]]) if axis = 2
+
+
+**Attributes**:
+
+* *reduction*
+
+  * **Description**: The type of operation to perform on the inputs.
+  * **Range of values**: one of ``copy``, ``sum``, ``prod``, ``min``, ``max``
```

Comment:
Is the value of `copy` an equivalent of ONNX `none` for this attribute?

Reply:
Yes, but can be renamed to "none" if preferred

Reply:
Done